Curtains/Curtains.Domain/Models/OrderModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class OrderModel
+    {
+        public OrderModel()
+        {
+        }
+
+        public int Id { get; set; }
+        public int UserId { get; set; }
+        public int PaymentTypeId { get; set; }
+        public int DeliveryTypeId { get; set; }
+        public string Comment { get; set; } = null!;
+
+        public virtual DeliveryTypeModel DeliveryType { get; set; } = null!;
+        public virtual UserModel User { get; set; } = null!;
+        public virtual PaymentTypeModel PaymentType { get; set; } = null!;
+
+        public virtual UserOrderModel UserOrder { get; set; } = null!;
```

Comment:
mb UserOrderModel here should be ICollection?

Reply:
No, because one user can have many orders, but one order can't be shared between many users.